.github/workflows/ci-windows.yml

```diff
@@ -104,4 +104,4 @@ jobs:
       run: ninja
     - name: Run AVIF Tests
       working-directory: ./build
-      run: ctest -j 2
+      run: ctest -j $Env:NUMBER_OF_PROCESSORS
```

Comment:
My experiments showed that this command is executed in PowerShell, not Command Prompt. So we need to use the PowerShell syntax `$Env:NUMBER_OF_PROCESSORS`, not the Command Prompt syntax `%NUMBER_OF_PROCESSORS%`.

The value of the environment variable `NUMBER_OF_PROCESSORS` is 2 in the virtual machine running the windows-latest os image.